### PR TITLE
tightvnc: Add version 2.8.27

### PR DESCRIPTION
**Pull request by digrouz**

### CODE REVIEW

**Comment by Ash258** [comment]
What is difference between tightvnc.net and tightvnc.com?

.com hash newer version listed and https compatible
.net is only http and have 2.8.11 version

**Comment by digrouz** [comment]
Applied changes

**Comment by digrouz** [comment]
I don't know :/

File: bucket/tightvnc.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "version": "2.8.27",
+    "description": "VNC-Compatible Remote Control / Remote Desktop",
+    "homepage": "http://tightvnc.net",
```
Comment (Ash258) [inline bucket/tightvnc.json]:
```suggestion
    "homepage": "https://tightvnc.com",
```

File: bucket/tightvnc.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    "extract_dir": "PFiles\\TightVNC",
+    "bin": [
+        "tvnviewer.exe",
+        "tvnserver.exe"
+    ],
+    "shortcuts": [
+        [
+            "tvnviewer.exe",
+            "TighVNC Viewer"
```
Comment (Ash258) [inline bucket/tightvnc.json]:
```suggestion
            "TightVNC Viewer"
```

File: bucket/tightvnc.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        ],
+        [
+            "tvnserver.exe",
+            "TighVNC Server"
```
Comment (Ash258) [inline bucket/tightvnc.json]:
```suggestion
            "TightVNC Server"
```

File: bucket/tightvnc.json (hunk 2 above)
Comment (digrouz) [reply bucket/tightvnc.json]: it is TightVNC isn't it ?

File: bucket/tightvnc.json (hunk 3 above)
Comment (digrouz) [reply bucket/tightvnc.json]: t is TightVNC isn't it ?

File: bucket/tightvnc.json (hunk 2 above)
Comment (Ash258) [reply bucket/tightvnc.json]: What do you think?

A

Comment (digrouz) [reply bucket/tightvnc.json]: indeed I made the mistake :p

**Comment by digrouz** [comment]
Thanks again for the support!

**Pull request merged**